omod/src/main/resources/config.xml

```diff
@@ -15,7 +15,16 @@
 	<activator>org.openmrs.module.atomfeed.AtomfeedActivator</activator>
 
 	<require_version>2.0.5</require_version>
-	
+
+	<require_modules>
+		<require_module>
```

Comment:
We should require also a particular version of a module. Example: [link](https://github.com/openmrs/openmrs-module-referencemetadata/blob/3cfce170f944fb84cab4b03d6b33cbc52bbfb44c/omod/src/main/resources/config.xml#L21)